streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -272,6 +273,7 @@ void retryWithBackoff(final Map<TaskId, Set<TopicPartition>> tasksToBeCreated, f
                 try {
                     Thread.sleep(backoffTimeMs);
                     backoffTimeMs <<= 1;
+                    backoffTimeMs = Math.min(backoffTimeMs, maxBackoffTimeMs);
```

Comment:
Hmm.. I was not aware of this, and could you point me to the un-statisfying behavior that user complain about? Would like to make sure if this is really an issue to fix before merging.

For this PR, I'd suggest we do not include it so that it can be merged.